### PR TITLE
feat(fonts): Remove useless fonts in iOS

### PR DESCRIPTION
**Pull request by chinesedfan**
Related to #402.

Picked out necessary fonts(FontAwesome, MaterialIcons and Octicons), according to 
[react-native-vector-icons docs](https://github.com/oblador/react-native-vector-icons#option-manually).

@machour I only have iOS development environment, hope you can tweak app.gradle as well.

### CODE REVIEW

File: ios/GitPoint.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -391,7 +346,9 @@
 		13B07FB71A68108700A75B9A /* main.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; name = main.m; path = GitPoint/main.m; sourceTree = "<group>"; };
 		146833FF1AC3E56700842450 /* React.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = React.xcodeproj; path = "../node_modules/react-native/React/React.xcodeproj"; sourceTree = "<group>"; };
 		1ABB0D8AB2424B1595DABB16 /* libRNPhotoView.a */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = archive.ar; path = libRNPhotoView.a; sourceTree = "<group>"; };
+		20A722EC74A74B50BD5DCCC9 /* Feather.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = Feather.ttf; path = "../node_modules/react-native-vector-icons/Fonts/Feather.ttf"; sourceTree = "<group>"; };
```
Comment (housseindjirdeh) [inline ios/GitPoint.xcodeproj/project.pbxproj]: Are we using `Feather` in the app as well? 🤔

**Comment by housseindjirdeh** [comment]
<3

Created an issue for Android #485

File: ios/GitPoint.xcodeproj/project.pbxproj (hunk 1 above)
Comment (chinesedfan) [reply ios/GitPoint.xcodeproj/project.pbxproj]: @housseindjirdeh No, we do not need `Feather.ttf`. I found those fonts also existed in group `Resources`, so another commit has just fixed it.

And I also tried to re-setup the whole environment, by `rm -rf node_modules && yarn unlink && yarn && yarn link`. Those fonts do not come back. 😄

**Comment by machour** [comment]
@chinesedfan thanks for taking care of this. Weird, I thought that `yarn link` was bringing back those fonts every time. I'll have a deeper look tonight

**Comment by machour** [comment]
@chinesedfan I just cloned the gitpoint repo, ran yarn && yarn run link, the fonts are back in ios:

```diff
diff --git a/ios/GitPoint.xcodeproj/project.pbxproj b/ios/GitPoint.xcodeproj/project.pbxproj
index 8c19e0d..db9ef16 100644
--- a/ios/GitPoint.xcodeproj/project.pbxproj
+++ b/ios/GitPoint.xcodeproj/project.pbxproj
@@ -5,7 +5,6 @@
 	};
 	objectVersion = 46;
 	objects = {
-
 /* Begin PBXBuildFile section */
 		00C302E51ABCBA2D00DB3ED1 /* libRCTActionSheet.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 00C302AC1ABCB8CE00DB3ED1 /* libRCTActionSheet.a */; };
 		00C302E71ABCBA2D00DB3ED1 /* libRCTGeolocation.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 00C302BA1ABCB90400DB3ED1 /* libRCTGeolocation.a */; };
@@ -53,6 +52,15 @@
 		B8D599F61D2A4797A2F71CAA /* libRNVectorIcons.a in Frameworks */ = {isa = PBXBuildFile; fileRef = AFDBB5669CAB4C5C8215B571 /* libRNVectorIcons.a */; };
 		DC5C35E71E37AD1800F3F526 /* FontAwesome.ttf in Resources */ = {isa = PBXBuildFile; fileRef = DC5C35DD1E37AD1800F3F526 /* FontAwesome.ttf */; };
 		EB0F176BE52B4561B9FF07AB /* libReactNativeConfig.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D7043BDB577E427391ECFBB0 /* libReactNativeConfig.a */; };
+		0F987BBB71C946DE89D05A69 /* Menlo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 7F22BFBD2496471D8FDFB605 /* Menlo.ttf */; };
+		556500AAC2514E6BB32849CF /* Entypo.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 3571300798E74AFE9F153904 /* Entypo.ttf */; };
+		C03DE9E1943B439C9C974E69 /* EvilIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = BB22020B52C94FDA9F606AF5 /* EvilIcons.ttf */; };
+		984CFE50B39A491D89B15AFB /* Feather.ttf in Resources */ = {isa = PBXBuildFile; fileRef = F3AE4422C40744309203680D /* Feather.ttf */; };
+		A561A666CCAA4F3092BBC675 /* Foundation.ttf in Resources */ = {isa = PBXBuildFile; fileRef = CEBCE47C33B341BE9226EE5A /* Foundation.ttf */; };
+		18E7CCD6C27244778FB86BC2 /* Ionicons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 1975C9EA6D544D698F4049D9 /* Ionicons.ttf */; };
+		47ACB843AF004287B620AA14 /* MaterialCommunityIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = B719612FD2824C5CA4A32A3D /* MaterialCommunityIcons.ttf */; };
+		3B5BC4EC0322436992710BDA /* SimpleLineIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 45188C172D194D79A5BCC0DF /* SimpleLineIcons.ttf */; };
+		E6F4307FF1A5443E9148EBA0 /* Zocial.ttf in Resources */ = {isa = PBXBuildFile; fileRef = A13263A5A5D54129A8C471A0 /* Zocial.ttf */; };
 /* End PBXBuildFile section */
 
 /* Begin PBXContainerItemProxy section */
@@ -377,6 +385,15 @@
 		ED1B22A89E134AA89E251577 /* Nunito-Light.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = "Nunito-Light.ttf"; path = "../src/assets/fonts/Nunito-Light.ttf"; sourceTree = "<group>"; };
 		EE0605281E45441288EA9435 /* RNDeviceInfo.xcodeproj */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = "wrapper.pb-project"; name = RNDeviceInfo.xcodeproj; path = "../node_modules/react-native-device-info/RNDeviceInfo.xcodeproj"; sourceTree = "<group>"; };
 		FBD0F4CBD299403498005E08 /* Nunito-SemiBold.ttf */ = {isa = PBXFileReference; explicitFileType = undefined; fileEncoding = 9; includeInIndex = 0; lastKnownFileType = unknown; name = "Nunito-SemiBold.ttf"; path = "../src/assets/fonts/Nunito-SemiBold.ttf"; sourceTree = "<group>"; };
+		7F22BFBD2496471D8FDFB605 /* Menlo.ttf */ = {isa = PBXFileReference; name = "Menlo.ttf"; path = "../src/assets/fonts/Menlo.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		3571300798E74AFE9F153904 /* Entypo.ttf */ = {isa = PBXFileReference; name = "Entypo.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/Entypo.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		BB22020B52C94FDA9F606AF5 /* EvilIcons.ttf */ = {isa = PBXFileReference; name = "EvilIcons.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/EvilIcons.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		F3AE4422C40744309203680D /* Feather.ttf */ = {isa = PBXFileReference; name = "Feather.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/Feather.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		CEBCE47C33B341BE9226EE5A /* Foundation.ttf */ = {isa = PBXFileReference; name = "Foundation.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/Foundation.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		1975C9EA6D544D698F4049D9 /* Ionicons.ttf */ = {isa = PBXFileReference; name = "Ionicons.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/Ionicons.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		B719612FD2824C5CA4A32A3D /* MaterialCommunityIcons.ttf */ = {isa = PBXFileReference; name = "MaterialCommunityIcons.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/MaterialCommunityIcons.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		45188C172D194D79A5BCC0DF /* SimpleLineIcons.ttf */ = {isa = PBXFileReference; name = "SimpleLineIcons.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/SimpleLineIcons.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		A13263A5A5D54129A8C471A0 /* Zocial.ttf */ = {isa = PBXFileReference; name = "Zocial.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/Zocial.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
 /* End PBXFileReference section */
 
 /* Begin PBXFrameworksBuildPhase section */
@@ -557,6 +574,15 @@
 				ED1B22A89E134AA89E251577 /* Nunito-Light.ttf */,
 				78A9FAE6B34B4FAC9F27CBC3 /* Nunito-Regular.ttf */,
 				FBD0F4CBD299403498005E08 /* Nunito-SemiBold.ttf */,
+				7F22BFBD2496471D8FDFB605 /* Menlo.ttf */,
+				3571300798E74AFE9F153904 /* Entypo.ttf */,
+				BB22020B52C94FDA9F606AF5 /* EvilIcons.ttf */,
+				F3AE4422C40744309203680D /* Feather.ttf */,
+				CEBCE47C33B341BE9226EE5A /* Foundation.ttf */,
+				1975C9EA6D544D698F4049D9 /* Ionicons.ttf */,
+				B719612FD2824C5CA4A32A3D /* MaterialCommunityIcons.ttf */,
+				45188C172D194D79A5BCC0DF /* SimpleLineIcons.ttf */,
+				A13263A5A5D54129A8C471A0 /* Zocial.ttf */,
 			);
 			name = Resources;
 			sourceTree = "<group>";
@@ -1233,6 +1259,15 @@
 				5DC6C311356B4973AE2599BF /* Nunito-Light.ttf in Resources */,
 				B66611A59CDF49A19C845B57 /* Nunito-Regular.ttf in Resources */,
 				4A0E0CE9909244398A873436 /* Nunito-SemiBold.ttf in Resources */,
+				0F987BBB71C946DE89D05A69 /* Menlo.ttf in Resources */,
+				556500AAC2514E6BB32849CF /* Entypo.ttf in Resources */,
+				C03DE9E1943B439C9C974E69 /* EvilIcons.ttf in Resources */,
+				984CFE50B39A491D89B15AFB /* Feather.ttf in Resources */,
+				A561A666CCAA4F3092BBC675 /* Foundation.ttf in Resources */,
+				18E7CCD6C27244778FB86BC2 /* Ionicons.ttf in Resources */,
+				47ACB843AF004287B620AA14 /* MaterialCommunityIcons.ttf in Resources */,
+				3B5BC4EC0322436992710BDA /* SimpleLineIcons.ttf in Resources */,
+				E6F4307FF1A5443E9148EBA0 /* Zocial.ttf in Resources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
diff --git a/ios/GitPoint/Info.plist b/ios/GitPoint/Info.plist
index fc2257e..9e25ee7 100644
--- a/ios/GitPoint/Info.plist
+++ b/ios/GitPoint/Info.plist
@@ -49,7 +49,7 @@
 		</dict>
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string></string>
+	<string/>
 	<key>UIAppFonts</key>
 	<array>
 		<string>FontAwesome.ttf</string>
@@ -60,6 +60,15 @@
 		<string>Nunito-Light.ttf</string>
 		<string>Nunito-Regular.ttf</string>
 		<string>Nunito-SemiBold.ttf</string>
+		<string>Menlo.ttf</string>
+		<string>Entypo.ttf</string>
+		<string>EvilIcons.ttf</string>
+		<string>Feather.ttf</string>
+		<string>Foundation.ttf</string>
+		<string>Ionicons.ttf</string>
+		<string>MaterialCommunityIcons.ttf</string>
+		<string>SimpleLineIcons.ttf</string>
+		<string>Zocial.ttf</string>
 	</array>
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
```

**Comment by chinesedfan** [comment]
@machour Oops. I checked by `yarn link` instead of `yarn run link`. New contributors will be confused about this.

**Comment by machour** [comment]
That's a completely different thing buddy :/

* `yarn link` symlinks a locale package so that you can use it during development
* `yarn run link` runs `react-native link` which links libraries/ressources used by the app (it's defined in our package.json)

**Pull request merged**